proto/proto/block_header.proto

```diff
@@ -23,5 +23,5 @@ message BlockHeader {
     // specifies the version of the protocol.
     uint32 version = 9;
     // the time when the block was created.
-    uint64 timestamp = 10;
+    fixed32 timestamp = 10;
```

Comment:
Let's keep `timestamp` as `uint64` for now.

Reply:
Are we going to store milliseconds in timestamp? Currently it seems to store seconds from Linux epoche, `u32` should be enough.